docs/ruleset_functions/baseline_systems/is_baseline_system_10.md

```diff
@@ -13,25 +13,23 @@
+8. does_hvac_system_serve_single_zone()  
+
  
 ## Logic:    
 - Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
 - Set is_baseline_system_10 = Not_Sys_10: `is_baseline_system_10 = "Not_Sys_10"`    
-- Check that there is one heating system, if there is then carry on: `if is_there_only_one_hvac_sys_heating_system(B_RMR, hvac_b.id) == TRUE:`   
-    - Check that there is no preheat system, if there is none then carry on: `if len(hvac_b.preheat_system) == Null or hvac_b.preheat_system[0].heating_system_type = "NONE" :`    
-        - Check if fansystem is constant volume, if yes then carry on: `if is_hvac_sys_fan_sys_CV(B_RMR, hvac_b.id) == TRUE:`  
-            - Check if the hvac system serves a single zone and that the zone only has one terminal unit: `if does_hvac_system_serve_single_zone(B_RMR, zone_id_list) == TRUE AND does_each_zone_have_only_one_terminal(B_RMR,zone_id_list) == TRUE:`     
-                - Check that the data elements associated with the terminal unit align with system 10: `if are_all_terminal_heat_sources_none_or_null(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_cool_sources_none_or_null(B_RMR,terminal_unit_id_list) == TRUE And are_all_terminal_fans_null(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_types_CAV(B_RMR,terminal_unit_id_list) == TRUE:`        
-                    - if coolingsystem is None and the heating type is a electric then Sys-10: `if is_hvac_sys_cooling_type_none(B_RMR, hvac_b.id) == TRUE AND is_hvac_sys_heating_type_elec_resistance(B_RMR, hvac_b.id) == TRUE: is_baseline_system_10 = "Sys-10"`  
+- Check that there is no preheat system, if there is none then carry on: `if len(hvac_b.preheat_system) == Null or hvac_b.preheat_system.heating_system_type == "NONE" :`    
+    - Check that there is no heating system, if there is none then carry on: `if len(hvac_b.heating_system) == Null or hvac_b.heating_system.heating_system_type == "NONE":`     
+        - Check that there is no cooling system if there is none then carry on: `is_hvac_sys_cooling_type_none(B_RMR, hvac_b.id) == TRUE`  
+            - Check that there is no fan system, if there is none then carry on: `if len(hvac_b.fan_system) == Null:`     
+                - Check if the hvac system serves a single zone and that the zone only has one terminal unit: `if does_hvac_system_serve_single_zone(B_RMR, zone_id_list) == TRUE AND does_each_zone_have_only_one_terminal(B_RMR,zone_id_list) == TRUE:`   
+                    - Check that the data elements associated with the terminal unit align with system 10: `if are_all_terminal_heat_sources_electric(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_cool_sources_none_or_null(B_RMR,terminal_unit_id_list) == TRUE And do_all_terminals_have_one_fan(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_types_CAV_With_None_Equal_to_Null(B_RMR,terminal_unit_id_list) == TRUE: is_baseline_system_10 = "Sys-10"`      
```

Comment:
@claperle and @mkarpman I agree with the changes you’ve made but the challenge is making the same change for system 9. We do not have furnace at terminal level for system 9 and the wee decided not to add it either.
I do think system 9 and system 10 should be consistently defined, whether it is at HVAC system level or terminal level.

Reply:
@supriyagoel I don't have strong feelings one way or the other.

Reply:
Allow either way. CML needs to revise RDS.